machine-learning/src/main.py

```diff
@@ -105,5 +96,17 @@ def run_engine(engine, path):
     return result
 
 
+def _get_model(model, task=None):
```

Comment:
Can this method be referenced above without an issue? (Before it is defined)

Reply:
Yes this works fine